.github/workflows/genisys-server.yml

```diff
@@ -48,6 +48,7 @@ jobs:
             - name: Build Meteor
               run: |
                 cd ${{ env.METEOR_DIR }}
+                meteor npm install
```

Comment:
I would recommend doing `meteor npm ci` for actions.

Reply:
Thanks, I'll try it out!